src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/navigate-modal/NavigateModal.scss

```diff
@@ -0,0 +1,80 @@
+@import 'src/styles/common';
+
+.browserNavBarControls {
+  margin: 15px 0 0 40px;
+
+  .navButtonWrapper {
+    display: inline-block;
+    margin-right: 35px;
+  }
+}
+
+.inputGroup {
+  margin-top: 10px;
+  height: 30px;
+  font-weight: $light;
+
+  label {
+    & > span {
+      display: inline-block;
+      margin-right: 10px;
+      text-align: right;
+      width: 32px;
+    }
+  }
+
+  input {
+    border: 1px solid $light-grey;
+    height: 30px;
+    padding: 2px 6px;
+  }
+}
+
+.coordInputs {
+  margin-top: 20px;
+
+  input {
+    width: 140px;
+  }
+}
+
+.regionInput {
+  margin-top: 30px;
+
+  input {
+    width: 230px;
+  }
+}
+
+.moveLeftButton,
+.moveRightButton {
+  height: 22px;
+  width: 22px;
+}
+
+.zoomInButton,
+.zoomOutButton,
+.browserReset {
+  height: 16px;
+  width: 16px;
+}
+
+.formButtons {
+  margin-top: 30px;
+  width: 272px; // TODO: find a better way to position the buttons rather than relying on the width
```

Comment:
Perhaps these two should be wrapped in a single element with a fixed width?

![image](https://user-images.githubusercontent.com/6834224/208412510-efe8b12f-7618-4c12-b7cd-c2f55e7ab011.png)

Or (possibly even better) perhaps the whole section can be set to have a certain fixed width?

![image](https://user-images.githubusercontent.com/6834224/208412661-7f729812-79a7-4310-8d78-ec3ff6294e88.png)

Reply:
Applied fixed width to the whole section ✅